src/js.rs

```diff
@@ -67,133 +67,132 @@ extern {
     pub fn eval(js_source_text: &str) -> Result<JsValue, JsValue>;
 }
 
-// Object.
+// Array
 #[wasm_bindgen]
 extern {
-    pub type Object;
-
-    /// The Object constructor creates an object wrapper.
-    ///
-    /// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object
-    #[wasm_bindgen(constructor)]
-    pub fn new() -> Object;
-
-    /// The `hasOwnProperty()` method returns a boolean indicating whether the
-    /// object has the specified property as its own property (as opposed to
-    /// inheriting it).
+    pub type Array;
+    /// The copyWithin() method shallow copies part of an array to another location in the same
```

Comment:
Nitpick: can you add a newline between `pub type Array` and the doc comment below it?